docs/persistentvolume-metrics.md

```diff
@@ -1,11 +1,11 @@
+| kube_persistentvolume_status_phase | Gauge | `persistentvolume`=&lt;pv-name&gt; <br>`phase`=&lt;Bound\                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |Failed\|Pending\|Available\|Released&gt;| STABLE |
+| kube_persistentvolume_claim_ref | Gauge | `persistentvolume`=&lt;pv-name&gt; <br>`claim_namespace`=&lt;<namespace>&gt; <br>`name`=&lt;<name>&gt;                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    | STABLE |
+| kube_persistentvolume_labels | Gauge | `persistentvolume`=&lt;persistentvolume-name&gt; <br> `label_PERSISTENTVOLUME_LABEL`=&lt;PERSISTENTVOLUME_LABEL&gt;                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       | STABLE |
```

Comment:
could you revert these changes? They seem unrelated to this PR and are breaking some part of the existing doc

Reply:
reverted!